src/Form/matching.js

```diff
@@ -256,7 +256,7 @@ class Matching {
                 // Any other input type is likely a false match
                 // Arguably "text" should be as well, but it can be used for password reveal fields
                 if (
-                    ['password', 'text'].includes(input.type) &&
+                    ['password'].includes(input.type) &&
```

Comment:
Shouldn't this be simply?

```suggestion
                    input.type === 'password' &&
```